<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_BAx">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BAx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_3"/>
   </widget>
   <widget class="QLabel" name="lbl_Minz">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Minz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BAy">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BAy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BAz">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BAz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BBx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BBx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Maxy">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Maxy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Maxz">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Maxz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BBy">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BBy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_1"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plot_2"/>
   </widget>
   <widget class="QLabel" name="lbl_Minx">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Minx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Maxx">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Maxx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Miny">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_Miny</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BBz">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_BBz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mBBx">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mBAx">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mnBBx">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mnBAx">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mBBy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mBAy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mnBBy">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mnBAy">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>lbl_Minz</zorder>
   <zorder>lbl_Miny</zorder>
   <zorder>lbl_Minx</zorder>
   <zorder>lbl_Maxz</zorder>
   <zorder>lbl_Maxy</zorder>
   <zorder>lbl_Maxx</zorder>
   <zorder>lbl_BBz</zorder>
   <zorder>lbl_BBy</zorder>
   <zorder>lbl_BBx</zorder>
   <zorder>lbl_BAz</zorder>
   <zorder>lbl_BAy</zorder>
   <zorder>lbl_BAx</zorder>
   <zorder>lbl_mnBAy</zorder>
   <zorder>lbl_mnBBy</zorder>
   <zorder>lbl_mBAy</zorder>
   <zorder>lbl_mBBy</zorder>
   <zorder>lbl_mnBAx</zorder>
   <zorder>lbl_mnBBx</zorder>
   <zorder>lbl_mBAx</zorder>
   <zorder>lbl_mBBx</zorder>
   <zorder>label</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
